code/modules/antagonists/heretic/items/heretic_armor.dm

```diff
@@ -0,0 +1,85 @@
+// Eldritch armor. Looks cool, hood lets you cast heretic spells.
+/obj/item/clothing/head/hooded/cult_hoodie/eldritch
+	name = "ominous hood"
+	icon_state = "eldritch"
+	desc = "A torn, dust-caked hood. Strange eyes line the inside."
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDESNOUT
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH | PEPPERPROOF
+	flash_protect = FLASH_PROTECTION_WELDER
+	clothing_traits = list(TRAIT_ALLOW_HERETIC_CASTING)
+
+/obj/item/clothing/head/hooded/cult_hoodie/eldritch/examine(mob/user)
+	. = ..()
+	if(!IS_HERETIC(user))
+		return
+
+	. += span_notice("Allows you to cast heretic spells while the hood is up.")
+
+/obj/item/clothing/suit/hooded/cultrobes/eldritch
+	name = "ominous armor"
+	desc = "A ragged, dusty set of robes. Strange eyes line the inside."
+	icon_state = "eldritch_armor"
+	inhand_icon_state = "eldritch_armor"
+	flags_inv = HIDESHOES|HIDEJUMPSUIT
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
+	allowed = list(/obj/item/melee/sickly_blade)
+	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/eldritch
+	// Slightly better than normal cult robes
+	armor = list(MELEE = 50, BULLET = 50, LASER = 50,ENERGY = 50, BOMB = 35, BIO = 20, FIRE = 20, ACID = 20)
+
+/obj/item/clothing/suit/hooded/cultrobes/eldritch/examine(mob/user)
+	. = ..()
+	if(!IS_HERETIC(user))
+		return
+
+	. += span_notice("Allows you to cast heretic spells while the hood is up.")
+
+// Void cloak. Turns invisible with the hood up, lets you hide stuff.
+/obj/item/clothing/head/hooded/cult_hoodie/void
```

Comment:
potentially make this a heretic focus while the hood is down (no stealthy casting?)

Reply:
I thought the purpose of the focus requirement was to give sec the ability to detain (non-rust) heretics, not to make stealthy spellcasting impossible (especially since an ambusher is probably gonna want to lead with eldritch grasp anyway, which doesn't requrie a focus).